src/stores/stores.service.ts

```diff
@@ -66,8 +66,15 @@ export class StoresService {
           row['가맹점명칭'],
         );
 
+        const count = await this.prismaService.store.count({
+          where: {
+            name: row['가맹점명칭'],
+            categoryCode: await this.nameToCode(row['업종']),
+          },
+        });
+
         try {
-          if (store) {
+          if (store && count === 0) {
```

Comment:
혹시 아래 코드도 같은 의미라면 아래 코드가 더 간결할 거 같아요!

```suggestion
          if (store && !count) {
```